decide/lic11.cpp

```diff
@@ -0,0 +1,30 @@
+#include <decide/decide.hpp>
+
+// There exists at least one set of two data points, (X[i],Y[i]) and
+// (X[j],Y[j]), separated by exactly G_PTS consecutive intervening points, such
+// that X[j] - X[i] < 0. (where i<j).
+// The condition is not met when NUMPOINTS <3.
+//
+// 1 <= G_PTS <= NUMPOINTS−2
+bool lic11(Points points, Parameters parameters) {
+  int npts = points.size();
+  int gpts = parameters.GPTS;
+
+  // The condition is not met if npts < 3.
+  if (npts < 3)
+    return false;
+
+  // Next point separated by kpts points.
+  for (int i = 0; i < npts - gpts; i++) {
+    Coordinate &first = points[i];
+    Coordinate &second = points[i + gpts];
```

Comment:
This is off-by-one if I understand correctly. _Separated_ by exactly gpts means `i + gpts + 1`, right?
This would also need to be fixed in the for loop condition.

Reply:
Resolved